js/adapt-background-selector.js

```diff
@@ -36,16 +33,15 @@ var BackgroundSelectorView = Backbone.View.extend({
     },
 
     setBackgroundImage: function () {
+      // Reset
+      this.image = "none";
       // Check device size
-      if (Adapt.device.screenSize === 'large' || Adapt.device.screenSize === 'medium') {
-        this.image = 'url('+this.model.get('_backgroundSelector')._src+')';
-      } else {
-        if(this.model.get('_backgroundSelector')._mobile._isEnabled){
-          this.image = 'url('+this.model.get('_backgroundSelector')._mobile._src+')';
-        }
-        if(this.model.get('_backgroundSelector')._hideOnMobile){
-          this.image = 'none';
-        }
+      if (Adapt.device.screenSize === 'large' && this.model.get('_backgroundSelector')._large._isEnabled) {
```

Comment:
Could be ` if (Adapt.device.screenSize === 'large' && this.model.get('_backgroundSelector')._large._src) {` instead of _isEnabled